pybliometrics/scopus/abstract_retrieval.py

```diff
@@ -799,16 +799,19 @@ def get_date(coverDate):
                 s += '\n   '.join([aff.name for aff in self.affiliation])
         
         elif self._view in ('REF'):
-            # Sort reference list by citationcount
-            top_n = 5
-            references = sorted(self.references, key=convert_citedbycount, reverse=True)
+            try:
+                # Sort reference list by citationcount
+                top_n = 5
+                references = sorted(self.references, key=convert_citedbycount, reverse=True)
 
-            top_references = [f'{reference.title} ({get_date(reference.coverDate)}). '+
-                              f'EID: {reference.id}' for reference in references[:top_n]]
+                top_references = [f'{reference.title} ({get_date(reference.coverDate)}). '+
+                                f'EID: {reference.id}' for reference in references[:top_n]]
 
-            s = f'A total of {self.refcount} references were found. '
-            s += f'Top {top_n} references:\n\t'
-            s += '\n\t'.join(top_references)
+                s = f'A total of {self.refcount} references were found. '
+                s += f'Top {top_n} references:\n\t'
+                s += '\n\t'.join(top_references)
+            except TypeError:
+                s = 'No references found'
 
```

Comment:
Can we change it such that is says "A total of 0 references were found."?
Ideally, the code becomes easier as well. In the try part, determine the number and the top references, in the except part set the number to 0 and the top references to empty.